core-plugins/src/e2e-test/features/fileplugin/sink/FileToFile.feature

```diff
@@ -0,0 +1,137 @@
+@File_Sink @FILE_SINK_TEST @FILE_SOURCE_TEST
+Feature:File Sink - Verification of File to File successful data transfer
+
+  Scenario Outline: To verify data is getting transferred from File to File sink successfully for different formats
+    Given Open Datafusion Project to configure pipeline
+    When Select plugin: "File" from the plugins list as: "Source"
+    When Expand Plugin group in the LHS plugins list: "Sink"
+    When Select plugin: "File" from the plugins list as: "Sink"
+    Then Connect plugins: "File" and "File2" to establish connection
+    Then Navigate to the properties page of plugin: "File"
+    Then Enter input plugin property: "referenceName" with value: "FileReferenceName"
+    Then Enter input plugin property: "path" with value: "csvFile"
+    Then Select dropdown plugin property: "format" with option value: "csv"
+    Then Click plugin property: "skipHeader"
+    Then Click on the Get Schema button
+    Then Verify the Output Schema matches the Expected Schema: "csvFileSchema"
+    Then Validate "File" plugin properties
+    Then Close the Plugin Properties page
+    Then Navigate to the properties page of plugin: "File2"
+    Then Enter input plugin property: "referenceName" with value: "FileReferenceName"
+    Then Enter input plugin property: "path" with value: "filePluginOutputFolder"
+    Then Replace input plugin property: "pathSuffix" with value: "yyyy-MM-dd-HH-mm-ss"
+    Then Select dropdown plugin property: "format" with option value: "<FileFormat>"
+    Then Validate "File" plugin properties
+    Then Close the Plugin Properties page
+    Then Save the pipeline
+    Then Preview and run the pipeline
+    Then Wait till pipeline preview is in running state
+    Then Open and capture pipeline preview logs
+    Then Verify the preview run status of pipeline in the logs is "succeeded"
+    Then Close the pipeline logs
+    Then Click on the Preview Data link on the Sink plugin node: "File"
+    Then Verify sink plugin's Preview Data for Input Records table and the Input Schema matches the Output Schema of Source plugin
+    Then Close the preview data
+    Then Deploy the pipeline
+    Then Run the Pipeline in Runtime
+    Then Wait till pipeline is in running state
+    Then Open and capture logs
+    Then Verify the pipeline status is "Succeeded"
+    Then Close the pipeline logs
+    Then Validate OUT record count is equal to IN record count
+    Then Validate output file generated by file sink plugin is equal to expected output file "<ExpectedOutputFile>"
+    Examples:
+      | FileFormat | ExpectedOutputFile    |
+      | csv        | outputForCSVInputTest |
+      | tsv        | tsvFormatOutput       |
+      | json       | jsonFormatOutput      |
+      | orc        | orcFormatOutput       |
+
+  Scenario: To verify data is getting transferred from File to File sink successfully for delimited format
```

Comment:
For delimited, csv & tsv formats, do we have tests for `Enable Quoted Values`?
![image](https://user-images.githubusercontent.com/88528384/189633975-d1b03110-9c57-4514-85c3-25321af92fc2.png)

Reply:
Added tests for `Enable Quoted Values`